Automattic/wp-calypso
# Jetpack Cloud: Remove 'jetpack/new-navigation' feature flag

This pull request removes the feature flag that gates whether or not to show the new sidebar navigation experience in Jetpack Cloud, and effectively defaults the app to behave as if this flag were enabled in all environments.

Resolves https://github.com/Automattic/jetpack-genesis/issues/88.

## Proposed Changes

* Remove the `jetpack/new-navigation` feature flag from all environment configs and default it to being "on."
* Remove all references to said feature flag in code.
* Update all style rules (e.g., `.is-jetpack-new-navigation`) and code paths to use the behavior that's defined when the feature flag is enabled.
* Remove all Jetpack Cloud sidebar components that are superseded by the new sidebar and no longer used.

## Testing Instructions

In your Jetpack Cloud test environment, explore all areas, _especially_ the sidebar navigation. Verify everything looks and behaves exactly as in production. If it helps, one approach to this is to have two tabs up (one in production, one in your test environment) and switch back and forth between them to look for differences.

If you'd like, you can also check in a Calypso Blue test environment that nothing is affected whatsoever, to verify this change doesn't cause problems there.
